pkg/pod/status.go

```diff
@@ -781,11 +781,12 @@ func extractContainerFailureMessage(logger *zap.SugaredLogger, status corev1.Con
 			}
 		}
 		if term.ExitCode != 0 {
-			return fmt.Sprintf("%q exited with code %d", status.Name, term.ExitCode)
+			// Include the termination reason
```

Comment:
Ideally the "why" is included in comments like this
```suggestion
			// Include the termination reason to add clarity for causes such as external signals, e.g. OOM
```